docs/tarball-readme.md

```diff
@@ -0,0 +1,11 @@
+# OpenTelemetry Collector for Google Cloud Monitoring Tarball
+
+Thank you for using this OpenTelemetry Collector and Google Cloud Monitoring. This packaged tarball contains tools to immediately setup the Collector with compatible metrics for your Google Cloud Monitoring.
+
+## Getting Started
+1. Run `bash run.sh` to run the collector
+2. Search for metrics in the Google Cloud Monitoring project, there should already be values populated!
```

Comment:
Is this true of the example config you include by default though? Should we create a full example config for Linux that actually generates all agent metrics?

Reply:
Yes. I am planning on having a config file with all the host metrics, and all the transformations needed and all of them prefixed with agent.googleapis. I am currently working on that config file.